integreat_cms/static/src/js/poi-box.ts

```diff
@@ -31,7 +35,12 @@ const hidePoiFormWidget = () => {
 
 const setPoi = ({ target }: Event) => {
     const option = (target as HTMLElement).closest(".option-existing-poi");
-    renderPoiData(option.getAttribute("data-poi-title"), option.getAttribute("data-poi-address"));
+    console.log(option);
```

Comment:
```suggestion
```

😁 

(Huh, eslint/prettier should in theory catch this, I have a look why it's not working)